gnucash/report/options-utilities.scm

```diff
@@ -83,12 +82,12 @@
    (gnc:make-multichoice-option
     pagename name-display-depth sort-tag help-string default-depth
     (list (vector 'all (N_ "All") (N_ "All accounts"))
```

Comment:
I think you missed this multi-choice option. It still has both a name and a description.